dove/src/tests_helper.rs

```diff
@@ -106,16 +109,23 @@ pub fn set_dependencies_local_move_stdlib(project_path: &Path) {
     .unwrap();
 }
 
+/// run dove
 pub fn execute_dove_at(args: &[&str], project_folder: &Path) -> anyhow::Result<()> {
     execute(args, project_folder.to_path_buf())
 }
-pub fn execute_dove_bin_at(args: &[&str], project_folder: &Path) -> anyhow::Result<String> {
+
+/// run bin dove
+pub fn execute_dove_bin_at(
+    args: &[&str],
+    project_folder: &Path,
+    cargo_bin_path: &str,
+) -> anyhow::Result<String> {
     assert!(
         project_folder.exists(),
         "Project folder {:?} does not exist",
         project_folder.display()
     );
-    let output = std::process::Command::new(env!("CARGO_BIN_EXE_dove"))
+    let output = std::process::Command::new(cargo_bin_path)
```

Comment:
What's the reason for such change?

Reply:
This variable is only visible in the tests folder